lib/spack/spack/compilers/intel.py

```diff
@@ -44,6 +46,13 @@ class Intel(Compiler):
                    'fc'  : 'intel/ifort' }
 
     @property
+    def openmp_flag(self):
+        if self.version < ver('16.0'):
+            return "-openmp"
+        else:
+            return "-qopenmp"
```

Comment:
Didn't realize `-openmp` was deprecated. Thanks for doing the research on this one.